mbuf/mbuf.go

```diff
@@ -43,6 +49,46 @@ func PktMbufClone(m *Mbuf, p *mempool.Mempool) {
 	C.rte_pktmbuf_clone(mbuf(m), mp(p))
 }
 
+// PktMbufAlloc allocate an uninitialized mbuf from mempool p.
+func PktMbufAlloc(p *mempool.Mempool) *Mbuf {
+	m := C.rte_pktmbuf_alloc(mp(p))
+	return (*Mbuf)(m)
+}
+
+// PktMbufAllocBulk allocate a bulk of mbufs.
+func PktMbufAllocBulk(p *mempool.Mempool, ms []*Mbuf) error {
+	return common.Err(C.rte_pktmbuf_alloc_bulk(mp(p), mbufs(ms), C.uint(len(ms))))
```

Comment:
Hmm, I forgot `common.Err` accepts an interface value, so there's an overhead if you call it in a hot path. Any suggestions?